test(formatter): Fix tests using TS syntax with `.js`

leaysgur opened the pull request:
As a result, it caused a parsing error, and nothing was recorded. 🙈 

- tests/fixtures/js/comments/yield.js
  - Removed, the same test already exists in ts/comments/yield.ts
- tests/fixtures/js/comments/assignments.js
  - Removed TS syntax
- tests/fixtures/ts/comments/assignments.ts
  - The same as original js/comments/assignment.js

Dunqing commented:
For the `yield` file, we should move it to the ts folder and rename it to `yield.ts`. This is what I want to test

Dunqing commented:
> For `yield.(j|t)s` case, I may find mismatches vs [Prettier & Biome](https://biomejs.dev/playground/?trailingCommas=none&arrowParentheses=as-needed&bracketSameLine=true&code=ZgB1AG4AYwB0AGkAbwBuACAAKgB0ADEAKAApACAAewAKACAAIAAgACAAeQBpAGUAbABkACAAKAAKACAAIAAgACAAIAAgACAAIAAvAC8AIABjAG8AbQBtAGUAbgB0AAoAIAAgACAAIAAgACAAIAAgAGEACgAgACAAIAAgACkAOwAKAH0ACgBmAHUAbgBjAHQAaQBvAG4AIAAqAHQAMgAoACkAIAB7AAoAIAAgACAAIAB5AGkAZQBsAGQAIAAoAAoAIAAgACAAIAAgACAAIAAgAC8ALwAgAGMAbwBtAG0AZQBuAHQACgAgACAAIAAgACAAIAAgACAAYQAKACAAIAAgACAAKQAgACsAIAAxADsACgB9AAoAZgB1AG4AYwB0AGkAbwBuACAAKgB0ADMAKAApACAAewAKACAAIAAgACAAeQBpAGUAbABkACAAKAAKACAAIAAgACAAIAAgACAAIAAvAC8AIABjAG8AbQBtAGUAbgB0AAoAIAAgACAAIAAgACAAIAAgAGEACgAgACAAIAAgACkAIAA%2FACAAMAAgADoAIAAxADsACgB9AAoAZgB1AG4AYwB0AGkAbwBuACAAKgB0ADQAKAApACAAewAKACAAIAAgACAAeQBpAGUAbABkACAAKAAKACAAIAAgACAAIAAgACAAIAAvAC8AIABjAG8AbQBtAGUAbgB0AAoAIAAgACAAIAAgACAAIAAgAGEACgAgACAAIAAgACkALgBiADsACgB9AAoAZgB1AG4AYwB0AGkAbwBuACAAKgB0ADUAKAApACAAewAKACAAIAAgACAAeQBpAGUAbABkACAAKAAKACAAIAAgACAAIAAgACAAIAAvAC8AIABjAG8AbQBtAGUAbgB0AAoAIAAgACAAIAAgACAAIAAgAGEACgAgACAAIAAgACkAWwBhAF0AOwAKAH0ACgBmAHUAbgBjAHQAaQBvAG4AIAAqAHQANgAoACkAIAB7AAoAIAAgACAAIAB5AGkAZQBsAGQAIAAoAAoAIAAgACAAIAAgACAAIAAgAC8ALwAgAGMAbwBtAG0AZQBuAHQACgAgACAAIAAgACAAIAAgACAAYQAKACAAIAAgACAAKQAoACkAOwAKAH0ACgBmAHUAbgBjAHQAaQBvAG4AIAAqAHQANwAoACkAIAB7AAoAIAAgACAAIAB5AGkAZQBsAGQAIAAoAAoAIAAgACAAIAAgACAAIAAgAC8ALwAgAGMAbwBtAG0AZQBuAHQACgAgACAAIAAgACAAIAAgACAAYQAKACAAIAAgACAAKQBgAGAAOwAKAH0ACgBmAHUAbgBjAHQAaQBvAG4AIAAqAHQAOAAoACkAIAB7AAoAIAAgACAAIAB5AGkAZQBsAGQAIAAoAAoAIAAgACAAIAAgACAAIAAgAC8ALwAgAGMAbwBtAG0AZQBuAHQACgAgACAAIAAgACAAIAAgACAAYQAKACAAIAAgACAAKQA7AAoAfQAKAGYAdQBuAGMAdABpAG8AbgAgACoAdAA5ACgAKQAgAHsACgAgACAAIAAgAHkAaQBlAGwAZAAgACgACgAgACAAIAAgACAAIAAgACAALwAvACAAYwBvAG0AbQBlAG4AdAAKACAAIAAgACAAIAAgACAAIABhAAoAIAAgACAAIAApADsACgB9AAoAZgB1AG4AYwB0AGkAbwBuACAAKgB0ADEAMAAoACkAIAB7AAoAIAAgACAAIAB5AGkAZQBsAGQAIAAoAAoAIAAgACAAIAAgACAAIAAgAC8ALwAgAGMAbwBtAG0AZQBuAHQACgAgACAAIAAgACAAIAAgACAAYQAKACAAIAAgACAAKQA7AAoAfQA%3D).
> 
> ```js
> // ORIGINAL
> function *t1() {
>     yield (
>         // comment
>         a
>     );
> }
> 
> // Oxc
> function* t1() {
>   yield (
>     // comment
>     a
>   );
> }
> 
> // Prettier
> function* t1() {
> 	yield // comment
> 	a;
> }
> 
> // Biome
> function* t1() {
> 	yield// comment
> 	a;
> }
> ```
> 
> I'm not sure it might just be a known issue, so I'll leave this to you. 🙏🏻

It is expected, and probably a bug in the `Prettier`.

leaysgur commented:
Removed. Now let me merge the stack. 🕺🏻

leaysgur commented:
### Merge activity

* **Oct 22, 8:20 AM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=oxc-project&repo=oxc) once it meets the requirements.

leaysgur committed:
test(formatter): Fix tests using TS syntax with `.js` (#14880)

As a result, it caused a parsing error, and nothing was recorded. 🙈

- tests/fixtures/js/comments/yield.js
  - Removed, the same test already exists in ts/comments/yield.ts
- tests/fixtures/js/comments/assignments.js
  - Removed TS syntax
- tests/fixtures/ts/comments/assignments.ts
  - The same as original js/comments/assignment.js
